### ISSUE TITLE
how to use switches in createProcess

### ISSUE THREAD

**Issue by cameronmcefee** [Adobe-CEP/CEP-Resources#187]
Hello!

Apologies if this is the wrong place for this sort of question. I'm trying to use the following command via createProcess:

```
wmic nic get adaptertypeid /format:csv
```

Based on the docs, it seems like this should be the trick:

```
window.cep.process.createProcess('/Windows/System32/wbem/WMIC.exe',
'nic',
'get',
'adaptertypeid',
'/format:csv')
```
However, wmi throws an error when I do this, saying it's an invalid query (the command works fine when run from the command line). Is there a special way I should be formatting the command so the switch doesn't break things?

If I omit the format switch it works:

```
window.cep.process.createProcess('/Windows/System32/wbem/WMIC.exe',
'nic',
'get',
'adaptertypeid')
```

Related, I would actually like to query multiple values like `adaptertypeid,macaddress,netconnectionstatus`, however that also breaks things.

**Comment by theiviaxx** [comment]
why did you close this?  Did you find a solution?

**Comment by cameronmcefee** [comment]
I opted for a different approach. From talking to a few other devs, using WMI creates permission related problems. Instead, I opted for adopting the approach [used here](https://github.com/automation-stack/node-machine-id#how-it-works), which thus far has been working fine in production, and worked with `createProcess`.

Be sure to make note of the [consideration of mixed vs native](https://github.com/automation-stack/node-machine-id/blob/master/index.js#L19). Since the CEP environment doesn't have a platform if you're not using Node, I opted for testing the userAgent string, which we can assume is untouched since we control the extension:

```javascript
const isMixed = function(){
  return /wow64/i.test(navigator.userAgent)
}
```
